### PR TITLE
chore(docker): add ability to change base href

### PR DESCRIPTION
**Pull request by udaraweerasinghege**
Fixes this; https://github.com/mozilla/fxa-dev/pull/321

### CODE REVIEW

File: bin/updateIndexFromEnv.js [hunk 1]
```diff
@@ -59,11 +60,16 @@ function applyEnvVars(currentConfig, envVars, serverConfig) {
   return currentConfig;
 }
 
-function writeNewConfigToIndexHtml(html, configToWrite) {
+function writeNewIndexHtml(html, configToWrite, baseUrl) {
   return new Promise((resolve, reject) => {
     const encodedConfig = encodeURIComponent(JSON.stringify(configToWrite));
     const newMetaTag = `<meta name="fxa-oauth-console/config/environment" content="${encodedConfig}" />`;
-    const newHtml = html.replace(META_TAG_PATTERN, newMetaTag);
+    var newHtml = html.replace(META_TAG_PATTERN, newMetaTag);
```
Comment (vladikoff) [inline bin/updateIndexFromEnv.js]: can we use `let` here?

File: docker-compose.yml [hunk 2]
```diff
@@ -9,6 +9,6 @@ services:
     ports:
       - "10137:10137"
     environment:
-      PROFILE_URI: https://127.0.0.1:9010/v1
+      PROFILE_URI: https://127.0.0.1:1111/profile/v1
```
Comment (vladikoff) [inline docker-compose.yml]: side nite: the profile uri looks wrong for local config. the content server repo or others have the correct one

Comment (vladikoff) [inline docker-compose.yml]: nit*

**Comment by udaraweerasinghege** [comment]
@vladikoff Fixed var -> let, had to use strict mode. 
The profile_url seems to be the same in content-server : https://github.com/mozilla/fxa-content-server/blob/c3cc9d4a462e9c96cd702a5e0161065c4d8c0139/server/config/local.json-dist#L5 && https://github.com/mozilla/fxa-content-server/blob/f9006d4e0042002c3f5624e6c626130601c8da22/server/lib/routes/get-openid-configuration.js#L12

**Comment by udaraweerasinghege** [comment]
So when i use `use strict;`(to use `let`), node is okay, but eslint fails because it doesn't want `use strict` in modules. But without `use strict` node wont run the file.

Should I add a new eslint rule to allow for use strict in modules, or just use `var`?

**Comment by udaraweerasinghege** [comment]
Changed eslint rule for now, let me know if you prefer to just use var.

**Comment by vladikoff** [comment]
> Changed eslint rule for now

That should be fine, thank you!

**Pull request merged**